src/main/java/seedu/address/ui/BookCard.java

```diff
@@ -91,7 +98,10 @@ public void setAuthor(Book book) {
                     .map(Author::toString)
                     .collect(Collectors.joining(", "));
             authors.setText("Author: " + authorsString);
+            return;
         }
+        authors.setVisible(false);
+        authors.setManaged(false);
```

Comment:
Based on the principle of "Making the happy path prominent", I think you should include the last two lines (103-104) in the if statement, and move the current lines in the if statement outside of the if statement

Reply:
Thanks for pointing out, will update in the upcoming PR 👍